crates/control/src/search_suggestor.rs

```
@@ -70,7 +71,7 @@ impl SearchSuggestor {
     }
 
     pub fn cycle(&mut self, mut context: CycleContext) -> Result<MainOutputs> {
-        self.update_heatmap(&context);
+        let _ = self.update_heatmap(&context);
```

why do you choose to silently ignore the error?

---

done